requirements.txt

```diff
@@ -10,3 +10,4 @@ setuptools
 ipython
 bumpversion
 twine
+sh
```

Comment:
You need to add this requirement to the "setup.py" file in order for the plug-in to work in production.

Reply:
Also, this Python module is bad ass! I never knew about it until now.